packages/url/src/index.ts

```diff
@@ -22,10 +22,11 @@ export type URLParams = {
   [key: string]: string
 }
 
-export const getPathname = (u: string): string => {
-  const end = u.indexOf('?')
+export const getPathname = (url: string): string => url.slice(0, getQueryIndex(url))
 
-  return u.slice(0, end === -1 ? u.length : end)
-}
+export const getQueryParams = (url = '/'): ParsedUrlQuery => parse(url.slice(getQueryIndex(url) + 1))
 
-export const getQueryParams = (url = '/'): ParsedUrlQuery => parse(url.slice(url.indexOf('?') + 1))
+const getQueryIndex = (url: string): number => {
```

Comment:
pla declare this function before using it

Reply:
Updated with new commit